version1/04 - Web services.md

```diff
@@ -364,6 +364,12 @@ Content-Type: application/json
         - field ID placeholder which identifies a business-specific metadata field
         - **values**
           - array of zero or more values to write to the specified business-specific metadata field
+  - we have screening inheritance logic build in the service(create only) for *Dokument*, *Registrering* and *Mappe* types, the future object screening is determined by rules of priority:
```

Comment:
Should `build` here be `built` instead?

Reply:
https://dictionary.cambridge.org/dictionary/english/build-in will leave it this way if no oposition?

Reply:
It should be `built-in` :) The reference you need is actually https://dictionary.cambridge.org/dictionary/english/built-in

Reply:
And since this is not a verb, but an adjective, you might even need another `in` afterwards. I suggest simplifying it so that we don't need to wonder about the wording in the first place :D : `we have screening inheritance logic as part of the create endpoints for ...`

Reply:
https://github.com/documaster/noark5-web-services/pull/59/commits/bf397c826b3799c67c5d9075250977e1041d2fde